src/components/Panel/PanelContainer.js

```diff
@@ -3,13 +3,18 @@
  * @copyright IBM Security 2019
  */
 
+/* eslint-disable no-useless-computed-key, react/require-default-props */
```

Comment:
What are these rules disabling?

Reply:
The proptypes block --

```jsx
  ['aria-label']: requiredIfGivenPropExists(
    'hasScrollingContent',
    PropTypes.string
  ),
```

-- contains both a computed property and a prop without a default

Reply:
Awesome! I wonder if we should set `no-useless-computed-key` as a global warning given that `requiredIfGivenPropExists`, seems like it could be a valuable helper method.